docs/FAQ/gui-faq-da.md

```diff
@@ -48,23 +48,24 @@ Den anden kategori kan med fordel kontakte deres DNS-udbyder med spørgsmål til
 testresultater, der ikke "lyser grønt" på egne domænenavne.
 
 #### <span id="q4"></span>4. Zonemaster rapporterer advarsler/fejl på mit domænenavn, hvad betyder det?
-Det kommer an på hvilke advarsler/fejl, der rapporteres. I de fleste tilfælde kan
-du trykke på den aktuelle "advarsler/fejl"-besked og få detaljerede
-informationer om problemet.
+Det kommer an på hvilke advarsler/fejl, der rapporteres.
+Hver test er ledsaget af en eller flere meddelelser, der beskriver de fundne problemer.
+Du kan også få yderligere indsigt om hver test fra dokumentet [Defined Test Cases] 
 
 #### <span id="q5"></span>5. Hvordan kan Zonemaster skelne mellem, hvad der er rigtigt og forkert?
```

Comment:
In the table of contents it says "Hvordan kan Zonemaster afgøre hvad der er rigtigt og forkert?".